templates/.github/workflows/automerge.yml

```diff
@@ -4,9 +4,32 @@
 on: [ pull_request ]
 
 jobs:
-  automerge:
+  automerge-check:
     if: github.event.pull_request.user.login == 'web3-bot'
     runs-on: ubuntu-latest
+    outputs:
+      status: ${{ steps.should-automerge.outputs.status }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
does this fetch all commits? I'd document that, because for a second I thought it meant zero commits.

Reply:
Yes, this is a non-shallow fetch. We need to do this, otherwise we don't know where `master` is.

Reply:
or rather, `github.event.pull_request.base.ref`